doc/quick_tutorial.rst

```diff
@@ -17,62 +17,108 @@
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
-.. _quick_tutorial_ref:
+.. Edited October 1, 2020 - [USA]TechDude
```

Comment:
Not should we should include edited dates (due to requiring us to always seek these places out when we edit these files in the future). GitHub will have our release history.

Reply:
We should include a link to the video tutorials here as well